test/postgrex_test.exs

```diff
@@ -1,30 +1,35 @@
 defmodule PostgrexTest do
   use ExUnit.Case, async: false
-  import ExUnit.CaptureLog
+  import ExUnit.CaptureLog, warn: false
 
   test "start_link/2 sets search path" do
-    # valid argument
     search_path = ["public", "extension"]
     {:ok, conn} = Postgrex.start_link(database: "postgrex_test", search_path: search_path)
     %{rows: [[result]]} = Postgrex.query!(conn, "show search_path", [])
+
     assert result == Enum.join(search_path, ", ")
+  end
 
-    # invalid argument
-    Process.flag(:trap_exit, true)
-    search_path = "public, extension"
+  # This test fails due to a bug betweem Elixir and Erlang in earlier versions of Elixir.
```

Comment:
I think there are two issues here:

1. Previous version did not log through Elixir logger
2. v1.17.0 and v1.17.1 broke the logging

```suggestion
  # gen_statem reports are only captured on Elixir v1.17+
  # but a bug causes the Logger to crash on v1.17.0 and v1.17.1.
```

Does this capture it fully?